tools/topology/topology2/cavs/CMakeLists.txt

```diff
@@ -17,13 +18,22 @@ PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt.bin"
 # TGL: CAVS HDA topology with mixer-based pipelines for HDA and passthrough pipelines for HDMI
 "cavs-passthrough-hdmi\;cavs-mixin-mixout-hda-2ch-tgl\;\
 HDA_CONFIG=mix,NUM_DMICS=2,PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt.bin"
+"cavs-passthrough-hdmi\;cavs-deepbuf-mixin-mixout-hda-2ch-tgl\;\
```

Comment:
no, this is becoming ridiculous to have a proliferation of names like this that the kernel doesn't use, which forces users to copy/override files.

This whole file is a mess - we don't want users to change defaults in an installed setup.

Reply:
sure, use the original name